soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -2145,7 +2145,6 @@ void FileChoose_LoadGame(GameState* thisx) {
     gSaveContext.seqId = (u8)NA_BGM_DISABLED;
     gSaveContext.natureAmbienceId = 0xFF;
     gSaveContext.showTitleCard = true;
-    gSaveContext.dogParams = 0;
```

Comment:
i'm assuming this is getting reset in enough other places that we don't actually need it here, but just from a "ensure authentic behavior changes are behind a cvar" perspective it might be worth throwing a `if (!CVar) {` check around it instead of just removing it